app/helpers/template_helpers.py

```diff
@@ -94,6 +94,14 @@ def service_links_context(
 
         return None
 
+    @property
+    def data_layer_context(
+        self,
+    ) -> Union[list[dict], list]:
```

Comment:
Returning `list` and `list[dict]` makes `list[dict]` redundant as `list[Any]` would supersede. It should just be `list[dict]`.

Reply:
Ok added.